react/PickupItems.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { FC } from 'react'
+
+import { useOrder } from './components/OrderContext'
+import ProductList from './components/ProductList'
+import StorePickUpHeader from './components/PickUpHeader'
+
+const PickupItemsList: FC = () => {
+  const { pickUpParcels } = useOrder()
+
+  if (pickUpParcels.length === 0) {
+    return null
+  }
+
+  return (
+    <section className="bb b--muted-4">
```

Comment:
Isn't this the OrderPlacedSection component?

Reply:
Nice catch 🤙